cmd/ndm_daemonset/controller/blockdevicestore.go

```diff
@@ -164,18 +164,21 @@ func (c *Controller) DeleteBlockDevice(name string) {
 		"ndm.blockdevice.delete.success", "Deleted blockdevice object ", name)
 }
 
-// ListBlockDeviceResource queries the etcd for the devices for the host/node
+// ListBlockDeviceResource queries the etcd for the devices
 // and returns list of blockdevice resources.
-func (c *Controller) ListBlockDeviceResource() (*apis.BlockDeviceList, error) {
+func (c *Controller) ListBlockDeviceResource(listAll bool) (*apis.BlockDeviceList, error) {
```

Comment:
n_; add a comment. listAll=true implies that all the block devices in the cluster will be returned. and if listAll=false, the block device on the node where NDM is running will be returned.

Reply:
Done.